doc/rtd/explanation/kernel-command-line.rst

```diff
@@ -2,18 +2,12 @@ Kernel command line
 *******************
 
 Providing configuration data via the kernel command line is somewhat of a last
-resort, since this method only supports
-:ref:`cloud config<user_data_formats-cloud_config>` starting with
```

Comment:
This piece of information doesn't apply to datasource discovery override, and is already mentioned in the section below that it is relevant to. Drop it.